source/Userdocs/Software/MatLab.md

```diff
@@ -1 +1,21 @@
-# MatLab
+(matlab)=
+# MatLab NeuroML Toolbox
```

Comment:
This hasn't seen a commit since 2018. Do we know if this is:

- still functional
- supported by someone using MatLab that can answer queries that may come up?